accord-core/src/test/java/accord/burn/BurnTest.java

```diff
@@ -311,6 +373,7 @@ static void burn(RandomSource random, TopologyFactory topologyFactory, List<Id>
 
         logger.info("Received {} acks, {} nacks, {} lost, {} truncated ({} total) to {} operations", acks.get(), nacks.get(), lost.get(), truncated.get(), acks.get() + nacks.get() + lost.get() + truncated.get(), operations);
         logger.info("Message counts: {}", messageStatsMap.entrySet());
+        logger.info("Took {} and in logical time of {}", Duration.ofNanos(System.nanoTime() - startNanos), Duration.ofMillis(queue.nowInMillis() - startLogicalMillis));
```

Comment:
this was mostly for me... I wanted to get a better grasp of how much "logical" time has passed as there is a lot of delayed logic